lntest/fee_service_test.go

```diff
@@ -0,0 +1,32 @@
+package lntest
+
+import (
+	"io/ioutil"
+	"net/http"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+)
+
+// TestFeeService tests the itest fee estimating web service.
+func TestFeeService(t *testing.T) {
+	service := startFeeService()
+	defer service.stop()
+
+	// Wait for service to start accepting connections.
+	time.Sleep(time.Second)
```

Comment:
This might not be enough for CI builds, can we use a wait predicate anywhere?

Reply:
Fixed it with `require.Eventually`